akka-stream/src/main/scala/akka/stream/impl/io/InputStreamPublisher.scala

```diff
@@ -87,8 +87,6 @@ private[akka] class InputStreamPublisher(is: InputStream, bytesReadPromise: Prom
 
   /** BLOCKING I/O READ */
   def loadChunk() = try {
-    val arr = Array.ofDim[Byte](chunkSize)
```

Comment:
This is *NOT* safe, see that the code uses ByteString1C which points to the array itself: https://github.com/akka/akka/blob/master/akka-actor/src/main/scala/akka/util/ByteString.scala#L106